NineChronicles.Headless/GraphTypes/ActionMutation.cs

```diff
@@ -129,6 +129,12 @@ public ActionMutation(NineChroniclesNodeService service)
                     {
                         Name = "consumableIds",
                         Description = "List of consumable id for use."
+                    },
+                    new QueryArgument<ListGraphType<RuneSlotInfoInputType>>
```

Comment:
If you don't want to `[null, null, ...]` as a `runeSlotInfos` argument, it should be like the below.

```suggestion
                    new QueryArgument<ListGraphType<NonNullGraphType<RuneSlotInfoInputType>>>
```